README.md

```diff
@@ -41,6 +41,12 @@ Most terminals or text displays / text editors using a monospace font (for examp
 go get github.com/rivo/uniseg
 ```
 
+## Testing
+
+```bash
```

Comment:
I don't think it is necessary to add this here. It doesn't provide much information about the repo. I would think that most people don't care about coverage. And if they do, they can look up in the Go docs how to do it.

So I think you should remove this change.

Reply:
Done!